common/src/main/java/us/myles/ViaVersion/api/boss/BossBar.java

```diff
@@ -80,13 +81,21 @@ public BossBar addPlayer(T player) {
     }
 
     /**
-     * Show the bossbar to a player (uuid)
+     * Show the bossbar to a player (uuid).
```

Comment:
Could maybe add a line on why there's adders for both uuid and connection (and why direct connection is preffered)